styles.css

```diff
@@ -0,0 +1,349 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Catamaran', sans-serif;
+}
+
+body {
+    background-image: url('img/guitar-blur.jpeg');
+    background-position: center;
+    background-attachment: fixed;
+    top:0;
+    font-size: 62.5%;
+    text-align: center;
+    color: rgb(240, 240, 210);
+}
+
+html {
+    scroll-behavior: smooth;
+}
+/* add conditional CSS for different*/
+
+h1 {
+    font-size: 5vw;
+}
+
+h2 {
+    font-size: 3rem;
+}
+
+h3 {
+    font-size: 2rem;
+}
+
+p {
+    font-size: 1rem;
+}
+
+li {
+    padding: 5px;
+    height: 40px;
+}
+
+ul {
+
+    list-style: none;
+    padding: 0;
+}
+
+a {
+    text-decoration: none;
+    color: rgb(240, 240, 210);
+    font-size: 1.7vw;
+}
+
+li a:hover {
+    font-size: 1.9vw;
+    -o-transition: 0.3s;
+    -webkit-transition: 0.3s;
+    transition: 0.3s;
+}
+
+a span {
+    display:none;
+}
+
+a:hover span {
+    display:inline;
+}
+
+.about-container {
+    /*float: left;*/
```

Comment:
Looks like this was commented to remove it - should be removed before merging with master

Reply:
Done